src/core/vertexmodel.h

```diff
@@ -350,6 +369,7 @@ class QFIELD_CORE_EXPORT VertexModel : public QAbstractListModel
     bool mHistoryTraversing = false;
 
     friend class VertexModelTest;
+    int mSnapToCommonAngleDegrees;
```

Comment:
Keep the friend class line at the very bottom of the class definition, and make sure you initialize your members (i.e. in this case int mSnapToCommonAngleDegrees = 0;